cmd/operator-sdk/cli/cli.go

```diff
@@ -86,6 +86,21 @@ func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
 	}
 	root.PersistentPreRun = rootPersistentPreRun
 
+	// Get the init command to hidden the plugin
+	var initCmd *cobra.Command
+	for _, cmd := range root.Commands() {
+		if cmd.Name() == "init" {
+			initCmd = cmd
+			break
+		}
+	}
+
+	// This wil hidden the plugins flag since it should not be visible to user
+	if initCmd != nil {
+		puginsFlag := initCmd.Flag("plugins")
+		puginsFlag.Hidden = true
+	}
```

Comment:
how about changing line 93 to:

```
cmd.Flag("plugins").Hidden = true
```

and then delete line 97-102?

Reply:
we need actually check if puginsFlag != nil

Reply:
even that additional checking can be done in the for loop on line 98, right?